swmmio/defs/config.py

```diff
@@ -3,11 +3,8 @@
 # This is the swmmio project root
 ROOT_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-# path to the SWMM5 executable used within the run_models module
-if os.name == 'posix':
-    SWMM_ENGINE_PATH = os.path.join(ROOT_DIR, 'lib', 'linux', 'swmm5')
-else:
-    SWMM_ENGINE_PATH = os.path.join(ROOT_DIR, 'lib', 'windows', 'swmm5_22.exe')
+# path to the Python executable used to run your version of Python
+PYTHON_EXE_PATH = "python"#os.path.join(os.__file__.split("lib/")[0],"bin","python")
```

Comment:
@aerispaha can you give me some advice here? This could break someone's run if they had python pointing to py2.7

Reply:
@aerispaha nope! If the user is not running inside an environment, this line just calls “python” rather than the version of python you are using to execute.